### PR TITLE
Feature/tandem search filters

### PR DESCRIPTION
**Pull request by scott062**
- Add ability to perform tandem search in mock where it accepts two sets of search params
- Add tandem_nbr to payload to differentiate between two result sets for tandem one and tandem two

### CODE REVIEW

File: src/services/availablepositions.js [hunk 1]
```diff
@@ -101,4 +149,26 @@ async function get_available_position_by_id(id) {
...
+    require: false,
+  })
+
+  const dataTandemTwo = await create_tandem_query(query, false, false).fetchPage({
+    withRelated: RELATED,
+    pageSize: query["request_params.page_size"] || 25,
+    page: query["request_params.page_index"] || 1,
+    require: false,
+  })
+
```
Comment (scott062) [inline src/services/availablepositions.js:166]: Just using existing logic for pagination - but it it fundamentally flawed for this request. I need to make two distinct queries from one set of request params(codes, bureaus... codes2, bureaus2... pageSize... ). The problem is that I cannot determine how many results to fetch for each distinct query without knowing the results of the other query, and without having a reasonable sort. For example, maybe we fetch the first 25 for both, then apply a sort on the 50 total results to return the first 25 sorted by location. Not sure yet

**Comment by scott062** [comment]
TO TEST: hit the mock at localhost:3333/positions/available/tandem and pass any ap params, and then pass an identical ap param with '2' appended. You should get back a payload with positions with the property tandem_nbr for respective results

File: src/services/availablepositions.js [hunk 2]
```diff
@@ -48,6 +49,53 @@ const formatData = data => {
...
+      d.bureau_code = bureau.bur
+      delete position.bureau
+      d.pos_skill_desc = skill.skill_descr
+      d.pos_skill_code = skill.skl_code
+      delete position.skill
+      d.ppos_capsule_descr_txt = capsuledescription.description
+      d.ppos_capsule_modify_dt = capsuledescription.last_modified
+      delete position.capsuledescription
+      d.tandem_nbr = isTandemOne ? 1 : 2
+      return { ...d, ...position }
```
Comment (mjoyce91) [inline src/services/availablepositions.js:94]: Let's add `cpn_desc` just to get something on a screen. Here is some notional data:

`"cpn_desc": "AF-Falls Church/Alexandria"`

File: src/services/common.js [hunk 3]
```diff
@@ -16,9 +16,53 @@ const FILTERS = {
...
+  "danger_pays2": { field: "positions.bt_danger_pay_num" },
+  "assign_cycles2": { field: "cycle_id" },
+  "location_codes2": { field: "positions.pos_location_code" },
+  "tod_codes2": { field: "positions.tod" },
+  "differential_pays2": { field: "positions.bt_differential_rate_num" },
+  "skills2": { field: "codes.skl_code" },
+  "cp_ids2": { field: "cp_id" },
+  "cps_codes2": { field: "cp_status" },
+  // "bid_seasons": { field: "bsn_id" },  -- NOT IN FSBID SWAGGER -- 
+  // "seq_nums": { field: "fv_seq_num" }, -- NOT IN FSBID SWAGGER --
```
Comment (mjoyce91) [inline src/services/common.js:52]: Are these comments just related to AP? I'm seeing them in swagger for FV

Comment (scott062) [reply src/services/common.js:52]: Yep my mistake - filters are for both so they should include all filters for pv/ap
